### PR TITLE
add city district

### PR DESCRIPTION
**Pull request by echoturing**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,3 +16,6 @@ main:
 clean:
 	-rm -fr $(GOPATH)
 	go clean -cache
+
+format:
```
Comment (lgarithm) [inline Makefile]: you can install golang extension in vscode, and turn on format on save.

File: cmd/chengdu/chengdu.go [hunk 2]
```diff
@@ -4,18 +4,48 @@ import (
...
 	"github.com/echoturing/buyhouse/linkhome"
+	"go.uber.org/zap"
 )
 
 var (
 	districts = []string{"gaoxin7", "tianfuxinqu"}
 )
 
 func main() {
+	logger, _ := zap.NewProduction()
```
Comment (lgarithm) [inline cmd/chengdu/chengdu.go]: All errors must be handled.
A common ways is 
```golang
if err != nil {
   log.Print(err)
   os.Exit(1)
}
```
or
```golang
if err != nil {
   glog.Exit(err)
}
```

**Comment by lgarithm** [comment]
did you force pushed?

**Comment by echoturing** [comment]
yes...
git commit --amend
git push -f

**Comment by lgarithm** [comment]
You can use squash merge later instead of force push.

**Pull request merged**